### ISSUE TITLE
Feature Request: RAG support for multimodal models

### ISSUE THREAD

**Issue by azaylamba** [aws-samples/aws-genai-llm-chatbot#612]
Currently, we can upload an image to get answers about it using multimodal models. But this feature doesn't support RAG workflow.
It would be good if the model first understand the image and then answers the question about it using the documents stored in the vector storage. This way, users would be able to get answers by uploading images of errors.

**Comment by charles-marion** [comment]
I agree with you, it would mean moving the multimodal logic (at least for bedrock) to the Langchain interface. (Or add Rag support toe the idefics one)

I will keep this issue updated if it's being added.

**Comment by maryamkhidir** [comment]
This is now supported in the [v5.0.0](https://github.com/aws-samples/aws-genai-llm-chatbot/releases/tag/v5.0.0) release.

**Comment by azaylamba** [comment]
@maryamkhidir This doesn't seem to be working in [v5.0.0](https://github.com/aws-samples/aws-genai-llm-chatbot/releases/tag/v5.0.0) release. Could you please check and confirm?
